transfer_nlp/plugins/trainers.py

```diff
@@ -297,5 +299,35 @@ def _inference(engine, batch):
 
         return engine
 
+    def finetune_last_layer(self):
```

Comment:
this is probably better named `replace_final_layer` or something since it's just setting things up, not actually fine tuning.